CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Change log itself follows [Keep a CHANGELOG](http://keepachangelog.com) format.
 
 ## Unreleased
 
+- Removed support for Elixir 1.4
```

Comment:
I guess should be under deprecated section

Reply:
I moved it to "Removed" since this is a removal and not a deprecation.